core/core.podspec

```diff
@@ -22,10 +22,6 @@ Pod::Spec.new do |spec|
         Alternatively, proper pod installation is performed during Gradle sync in the IDE (if Podfile location is set)"
     end
                 
-    spec.xcconfig = {
-        'ENABLE_USER_SCRIPT_SANDBOXING' => 'NO',
-    }
```

Comment:
Could you provide more context about this change? I'm guessing this will leave the setting up to the consumer right?

Reply:
I honestly don't have any context. These were automatically regenerated, so I didn't make these changes myself. (See PR description 😛) 

Can revert, but we may be "fighting" the code generation at that point.

Reply:
Ok from what I read, I think this should be fine. We can always try to revert if people have issues 👍

Reply:
Thanks for checking! 🙏